wesnoth-multiplayer.po

```diff
@@ -2083,6 +2072,11 @@ msgid ""
 "perilous and unstable, is nonetheless rich thanks to numerous foundries "
 "using the heat of volcanoes and hot springs to fuel their forges."
 msgstr ""
+"Západní část Srdcových hor vytváří zemi pozoruhodných kontrastů – vroucí "
+"sopečné kotle tkající temně modré řeky, proplétající se mezi špičatými "
```

Comment:
dumal jsem o tom, zda mezi těmi slovesnými jmény má být čárka, ale navrhuji to raději rozštípnout
```suggestion
"sopečné kotle tkající temně modré řeky, jež se proplétají mezi špičatými "
```